react/react1/week1/todos-app/src/components/ToDoList.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'; 
+import ToDoItem from './ToDoItem';
+
+export default function ToDosList ({todoInfo}) {
+    return (
+        <div>
+            {todoInfo.map((list, index) => (
+                <ToDoItem key={index} description={list.description} deadline={list.deadline}/>
```

Comment:
I guess you're trying to pass an item, so a "list" wouldn't be an appropriate naming for the parameter.
Also, you could pass for example item={item} because it already contains the description and the deadline inside. :)

Reply:
Hi @annamourasobral  
Thank you so much for your review 
if I have change list to item and also modify 
  <ToDoItem key={index} description={list.description} deadline={list.deadline}/> to
    <ToDoItem key={index} item{item}/> 
    meaning that in my ToDoItem.jsx when I have to refer props need to access like 
    item.description, and item.deadline right?
    
    
    `export default function ToDoItem ({item}) {
   return (
    <p>
      {item.description} | {item.deadline}
    </p>
   )
}`

Reply:
Hi @annamourasobral  
Thank you so much for your review 
if I have change list to item and also modify 
  <ToDoItem key={index} description={list.description} deadline={list.deadline}/>
   to
    <ToDoItem key={index} item{item}/> 
    
    meaning that in my ToDoItem.jsx when I have to refer props need to access like 
    item.description, and item.deadline right?
    
 ```
export default function ToDoItem ({item}) {
   return (
    <p>
      {item.description} | {item.deadline}
    </p>
   )
}

```

Reply:
Precisely!
I know it might sound crazy because you just changed the component you're accessing the prop, but I have 2 main reasons to ask for this small modification:
1 - You don't really need the item description and deadline on the `TodoList.jsx`
2 -  In the future, you'll treat much bigger and more complex objects, and passing the props destructured as you were, could make the code messy.   
For example:
I have the object anna:
```
const anna = {
  first_name: "Anna",
  last_name: "Sobral",
  profession: "Frontend Engineer",
  age: 26,
  hobbies: ["play Nintendo switch", "hiking", "read a book"],
  diary:[{
  date: 24-02-2023,
  entry: "Today I reviewed @Kositthai code, she's awesome!"
  },{
  date: 25-02-2023,
  entry: "Today I replied @Kositthai's comment in her PR, she's very interested!"
  }]
}
```
this is a very small object, but can you imagine how many props you could make out of it?
so by just using anna={anna} in a component tree, you would pass everything at once. 
I hope this was useful and if you need anything, just pm me on slack! :)

Reply:
I didn't see the picture before because we're working with 2 keys-values object 
Your explanation absolutely makes sense!  

I have fixed my code, let me know if I can merge it or if I have to adjust something else 
Thank you so much for your time and feedback :)